portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,105 @@
+				upstate New Mexico in a ~spooky~ castle (2 years). I currently study Computer Science and try whenever
+				possible to connect it with all my other interests—language learning, music, design, and (attempted)
+				productivity.
+			</p>
+		</div>
+		<div id="projects">
+			<h2>My (Presentable) Code: </h2>
+			<div id="proj-bttns">
+				<button class="p-button" onclick="filterProjects('all')">All</button>
+				<button class="p-button" onclick="filterProjects('lang')">Human Languages</button>
+				<button class="p-button" onclick="filterProjects('js')">Javascript</button>
+				<button class="p-button" onclick="filterProjects('react')">React</button>
+				<button class="p-button" onclick="filterProjects('ts')">TypeScript</button>
+				<button class="p-button" onclick="filterProjects('sass')">Sass</button>
+				<button class="p-button" onclick="filterProjects('jekyll')">Jekyll</button>
+			</div>
+			<div class="project-desc">
+				<div id="word-adder" class="lang js">
+					<a href="https://github.com/chiazo/word-adder">Word Adder</a>
+					<div>
+						<p>A program to update foreign vocab lists hosted on google sheets with definitions & example
+							sentences</p>
+					</div>
+				</div>
+				<div id="my-planner" class="lang react js">
+					<a href="https://chiazo.github.io/my-planner/">My Planner</a>
+					<div>
+						<p>A web app that creates a daily schedule based on inputed tasks and their estimated time for
+							completion.</p>
+					</div>
+				</div>
+				<div id="read-learn" class="lang ts">
+					<a href="https://chiazo.github.io/read-learn">Read Learn</a>
+					<div>
+						<p>A web app that takes in a text in another language, allowing the user to click words they
+							don't know. Outputs a list of words to study later!</p>
+					</div>
+				</div>
+				<div id="syndrome" class="css js sass">
+					<a href="https://chiazo.github.io/syndrome/">Syndrome</a>
+					<div>
+						<p>A CSS framework built using Sass. Syndrome is a toolkit featuring all of my favorite CSS
+							styles in one place.</p>
+					</div>
+				</div>
+				<div id="personal-site" class="css js jekyll">
+					<a href="https://chiazo.github.io/">chiazo.github.io</a>
+					<div>
+						<p>My personal website!</p>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="gallery">
+			<h2>My Photography</h2>
+			<div id="photography">
+				<img src="images/hk2.jpg" class="photos"/>
+				<img src="images/hk5.jpg" class="photos"/>
+				<img src="images/hk7.jpg" class="photos"/>
+				<img src="images/hk8.jpg" class="photos"/>
+		</div>
+			</div>
+			<div class="links">
+				<ul>
+					<li><i><a href="https://www.linkedin.com/in/chiazo"><img src="/images/favicons/linkedin.png"/></a></i>
+					</li>
+					<li><i><a href="https://github.com/chiazo"><img src="/images/favicons/github.png"/></a></i></li>
+					<li><i><a href="https://chiazo.github.io"><img src="/images/favicons/website.png"/></a></i></li>
+				</ul>
+			</div>
+		</div>
```

Comment:
This bit here could use a run of the code formatter.

Reply:
Ok, I've formatted it again!